gdx/src/com/badlogic/gdx/graphics/g2d/Animation.java

```diff
@@ -75,6 +75,15 @@ public Animation (float frameDuration, Array<? extends T> keyFrames, PlayMode pl
 		setPlayMode(playMode);
 	}
 
+	/** Constructor, storing the frame duration and key frames.
+	 *
+	 * @param frameDuration the time between frames in seconds.
+	 * @param keyFrames the objects representing the frames.
+	 * return the correct type of array. Otherwise, it returns an Object[].*/

```

Comment:
This javadoc doesn't make sense.

Reply:
I think he copied the return from the constructor above with the same weird return text
https://github.com/libgdx/libgdx/blob/7c2c9b01e9e76379dea9c2720f7a566ae3b959da/gdx/src/com/badlogic/gdx/graphics/g2d/Animation.java#L68-L76

Reply:
I did